content/zh-cn/docs/concepts/storage/volume-snapshots.md

```diff
@@ -291,6 +291,11 @@ source volume's mode is unknown.
 `sourceVolumeMode` 是创建快照的卷的模式。`sourceVolumeMode` 字段的值可以是
 `Filesystem` 或 `Block`。如果没有指定源卷模式，Kubernetes 会将快照视为未知的源卷模式。
 
+<!--
+`volumeSnapshotRef` is the reference of the corresponding `VolumeSnapshot`. Note that when the `VolumeSnapshotContent` is being created as a pre-provisioned snapshot, the `VolumeSnapshot` referenced in `volumeSnapshotRef` might not exist yet.
+-->
+`volumeSnapshotRef` 字段是对相应的 `VolumeSnapshot` 的引用。请注意，当 `VolumeSnapshotContent` 被创建为预配置快照时。`volumeSnapshotRef` 中引用的 `VolumeSnapshot` 可能还不存在。
```

Comment:
```suggestion
`volumeSnapshotRef` 字段是对相应的 `VolumeSnapshot` 的引用。
请注意，当 `VolumeSnapshotContent` 被创建为预配置快照时。
`volumeSnapshotRef` 中引用的 `VolumeSnapshot` 可能还不存在。
```
Please read through [localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%8E%92%E7%89%88%E6%A0%BC%E5%BC%8F), remember it, and show your professional in every localized PR whatever the en upstream is bad or good, which might be the root to get respected.